### PR TITLE
Errors pkg

### LINKED ISSUE #19: Create an errors package to define all errors
Refactor currently existing errors

### PR DESCRIPTION
**Pull request by MarwanGalal746**

### CODE REVIEW

File: pkg/errs/errors.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package errs
+
+import "errors"
+
+var DbError = errors.New("unexpected database error")
+var ContentType404 = errors.New("content type not found")
+var Column404 = errors.New("column not found")
+var Content404 = errors.New("content not found")
```
Comment (odetolakehinde) [inline pkg/errs/errors.go]: have this as 
`var (`
 `// the errors go here`
`)`

File: pkg/errs/errors.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package errs
+
+import "errors"
+
+var DbError = errors.New("unexpected database error")
+var ContentType404 = errors.New("content type not found")
+var Column404 = errors.New("column not found")
```
Comment (odetolakehinde) [inline pkg/errs/errors.go]: You might want to name these errors with an `Error` or `Err` prefix like so: `ErrorColumnNotFound`

File: pkg/errs/errors.go (hunk 1 above)
Comment (MarwanGalal746) [reply pkg/errs/errors.go]: Done, check the last commit

File: pkg/errs/errors.go (hunk 2 above)
Comment (MarwanGalal746) [reply pkg/errs/errors.go]: Done, check the last commit

**Review by odetolakehinde — APPROVED** [review]

**Pull request merged**